data/registry/instrumentation-dotnet-kafkaflow.yml

```diff
@@ -16,7 +16,7 @@ authors:
 urls:
   repo: https://github.com/Farfetch/kafkaflow/tree/master/src/KafkaFlow.OpenTelemetry
 createdAt: 2023-12-07
-isFirstParty: true
+isFirstParty: false
```

Comment:
Can you revert this change? This is firstParty

Reply:
Done!